www/admin/artworks/post.php

```diff
@@ -0,0 +1,40 @@
+<?
+require_once('Core.php');
+
+if(HttpInput::RequestMethod() != HTTP_PATCH){
```

Comment:
More of a question than a request that this be changed: what is the purpose of the `_method` field and the check for `HTTP_PATCH` here?

I understand that it's a mechanism that already existed in the codebase, but it's not used anywhere else as far as I can tell. I suppose it indicates intent, but it's somewhat confusing at first glance since the actual request is still `POST` (and anyone seeing `PATCH` might assume that the request is being made by something other than an HTML form).